apps/api/migrations/normalize-message-template-cta-action/normalize-message-template-cta-action-migration.spec.ts

```diff
@@ -1,5 +1,5 @@
 import { expect } from 'chai';
-import { faker } from '@faker-js/faker';
+import faker from '@faker-js/faker';
```

Comment:
I don't think this should have changed, right?

```suggestion
import { faker } from '@faker-js/faker';
```

Reply:
true it was created like this by the idea, i will revert it.
thanks for noticing!!

Reply:
@djabarovgeorge we still have a number of occurrences of these changes we need to revert.

Reply:
I think we are ok. i updated those cases.